demo/admin/src/products/ManufacturersGrid.tsx

```diff
@@ -40,7 +41,8 @@ function ManufacturersGridToolbar() {
             </ToolbarItem>
             <FillSpace />
             <ToolbarItem>
-                <Button startIcon={<AddIcon />} component={StackLink} pageName="add" payload="add" variant="contained" color="primary">
+                {/* @ts-expect-error TODO: Fix component-prop in Button */}
```

Comment:
Will be fixed in a follow-up PR. 
I added a TODO item to the feature branch PR: https://github.com/vivid-planet/comet/pull/3062